src/index.html

```diff
@@ -6,9 +6,50 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="./images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav class="header__navigation">
+        <ul class="navigation__list">
+          <li class="list__item">
+            <a href="#" class="is-active item">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="item">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="item">Smartphones</a>
+          </li>
+          <li class="list__item">
+            <a
+            href="#"
+            class="item"
+            data-qa="hover"
+            >
```

Comment:
please fix the code styling according to the checklist point
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/f9cf70ab-6883-4ee9-b193-7e08164ef490)